certbot/cli.py

```diff
@@ -677,7 +666,7 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         "e.g. -vvv.")
     helpful.add(
         None, "-t", "--text", dest="text_mode", action="store_true",
-        help="Use the text output instead of the curses UI.")
+        help="Use the text output.")
```

Comment:
Oh right now I remember, I left this in on the off chance we eventually implement something else for the `IDialog` interface. I'll make this clear though.